### DISCUSSION TITLE
请问大家如何解决的TS获取多环境中的参数代码飘红~后端不太懂

### DISCUSSION THREAD

**Discussion by Jixiangup** [umijs/umi#7382] [Q&A] [answered]
console.log(AUTH_DOMAIN) 就是这样 这个`AUTH_DOMAIN`在读取`config.dev.ts`中的值运行浏览器是有值的，但是代码会飘红。。。

**Comment by leftstick** [comment, marked as answer]
项目根目录下有个`typings.d.ts` 看到没? 里面加一段：

```ts
declare var AUTH_DOMAIN: string
```

你代码里引用`AUTH_DOMAIN` 就不会红线了

**Reply by Jixiangup** [reply]
谢谢

**Comment by xiaohuoni** [comment]
`// @ts-ignore` yyds

**Reply by ideaviewes** [reply]
// @ts-ignore的话，还用什么ts
